pkg/kratos/service_test.go

```diff
@@ -0,0 +1,635 @@
+package kratos
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+	"fmt"
+	"io"
+	"net/http"
+	"net/http/httptest"
+	reflect "reflect"
+	"testing"
+
+	"github.com/canonical/identity-platform-login-ui/internal/monitoring"
+	gomock "github.com/golang/mock/gomock"
+	hClient "github.com/ory/hydra-client-go/v2"
+	kClient "github.com/ory/kratos-client-go"
+	"go.opentelemetry.io/otel/trace"
+)
+
+//go:generate mockgen -build_flags=--mod=mod -package kratos -destination ./mock_logger.go -source=../../internal/logging/interfaces.go
+//go:generate mockgen -build_flags=--mod=mod -package kratos -destination ./mock_kratos_pkg.go -source=./interfaces.go
```

Comment:
```suggestion
//go:generate mockgen -build_flags=--mod=mod -package kratos -destination ./mock_interfaces.go -source=./interfaces.go
```

simply to avoid having the package name in here, I should have done it in here https://github.com/canonical/identity-platform-login-ui/blob/main/pkg/extra/handlers_test.go#L19 too